homeassistant/components/telegram_bot/__init__.py

```diff
@@ -951,7 +950,7 @@ def __init__(self, hass, config):
         self.hass = hass
 
     def handle_update(self, update: Update, context: CallbackContext) -> bool:
```

Comment:
I assume the `handle_update` will be called inside the event loop? Then we need to use the async API inside. Eg use `hass.bus.async_fire`.